central/reports/service/v2/convert.go

```diff
@@ -82,7 +83,10 @@ func (s *serviceImpl) convertV2VulnReportFiltersToProto(filters *apiV2.Vulnerabi
 	ret := &storage.VulnerabilityReportFilters{
 		Fixability:       storage.VulnerabilityReportFilters_Fixability(filters.GetFixability()),
 		AccessScopeRules: accessScopeRules,
-		IncludeNvdCvss:   filters.GetIncludeNvdCvss(),
+	}
+	if features.NvdCvss.Enabled() {
```

Comment:
These are sort of buried in the converter.  Are you sure these are the only places requiring change?